R/card.R

```diff
@@ -0,0 +1,298 @@
+#' Card grid
+#'
+#' Use to create a grid of cards, giving every card in the entire grid the same
+#' height/width (or optionally, every card in _each row_ of the grid).
+#'
+#' @param ... Named arguments become attributes on the `<div class="card">`
+#'   element. Unnamed arguments are children, and should be instances of
+#'   [card()].
+#' @param class Additional CSS classes to include on the grid div.
+#' @param card_width The desired width of each card. This can be any [CSS length
+#'   unit][htmltools::validateCssUnit()], in which case `card_grid` will fit as
+#'   many cards into each row without being smaller than that width.
+#'   Alternatively, you can specify a number of cards per row by passing
+#'   `1/num`, for example `card_width = 1/3` for three cards per row. You can
+#'   also set this to `NULL` and set the `grid-template-columns` CSS property
+#'   manually, either via a `style` attribute or a CSS stylesheet.
+#' @param gap A [CSS length unit][htmltools::validateCssUnit()] that sets the
+#'   amount of spacing between cards. If `NULL`, the value of Bootstrap's
+#'   `$spacer` Sass variable is used.
+#' @param heights_equal If `"all"` (the default), every card in every row of the
+#'   grid will have the same height. If `"row"`, then every card in _each_ row
+#'   of the grid will have the same height, but heights may vary between rows.
+#' @param fixed_width When `card_width` is a CSS length (as opposed to a
+#'   fraction), the default behavior is to let cards be that wide _or larger_,
+#'   so that they fill all of the horizontal space that's available. Set this
+#'   argument to `TRUE` to ensure that cards are exactly the specified width,
+#'   even if this means using less than the total width of the card grid
+#'   container.
+#'
+#' @export
```

Comment:
We should also include `@example` and/or link to the article

```suggestion
#' @export
#' @family cards
```